charts/base/templates/pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.storage -}}
```

Comment:
please add commented sample config in charts/base/values.yaml file for this resource